knowledge-base/htmltextbox-an-error-occurred-while-parsing-entityname.md

```diff
@@ -0,0 +1,37 @@
+---
+title: '#ERROR# An error occurred while parsing EntityName'
+description: 'An error is thrown while parsing the EntityName of HtmlTextBox using special characters in its value'
+type: how-to
+page_title: Parsing the HtmlTextBox value leads to an EntityName error
+slug: htmltextbox-an-error-occurred-while-parsing-entityname
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting HtmlTextBox</td>
```

Comment:
There is no such product. The product is Telerik Reporting, while the HtmlTextBox is a report element. Change the Table accordingly.